EIPS/eip-5345.md

```diff
@@ -0,0 +1,133 @@
+---
+eip: 5345
+title: Walletconnect Silent-Signing Extension
+description: Capability of transaction signing without user interaction for a period of time for walletconnect
+author: Stanley Wu (@fruit37), Mücahit Büyükyılmaz (@anndro), Muhammed Emin Aydın (@muhammedea)
+discussions-to: https://ethereum-magicians.org/t/walletconnect-silent-signing-extension/10137
+status: Draft
+type: Standards Track
+category: Interface
```

Comment:
Note this is already in the `interface` track, and shoudl probably be adjusted accordingly to describe functionality for ALL wallets